src/to-contain-element.js

```diff
@@ -1,9 +1,16 @@
-import {matcherHint, printReceived} from 'jest-matcher-utils'
+import {
+  matcherHint,
+  stringify,
+  RECEIVED_COLOR as receivedColor,
+} from 'jest-matcher-utils'
 import {checkHtmlElement} from './utils'
 
 export function toContainElement(container, element) {
   checkHtmlElement(container, toContainElement, this)
-  checkHtmlElement(element, toContainElement, this)
+
+  if (element) {
```

Comment:
Using a truthy check like this will mean these test cases will pass:

```js
expect(container).not.toContainElement(false);
expect(container).not.toContainElement(0);
expect(container).not.toContainElement("");
```

Is that the behaviour we want? (@gnapse)

Reply:
Hrm, awesome point!

`element.contains` should throw a TypeError for those items (false, 0, and '') instead of returning false.

```
E.G.:
document.body.contains(0)
document.body.contains(false)
document.body.contains()
```

Reply:
@jgoz should we intentionally throw an error if it is not `null`, `undefined` or `HTMLElement`?

Something like:

`toBeInTheDocument expects an HTMLElement, null, or undefined but received 0`

Reply:
Kk, running through a test on jsdom, it appears it throws errors like chrome and FF:

```
test('if `element.contains` throw an error for falsey values', () => {
	expect(() => document.documentElement.contains(0)).toThrowError();
	expect(() => document.documentElement.contains('')).toThrowError();
	expect(() => document.documentElement.contains(false)).toThrowError();
});
```

Reply:
I was thinking it could fall through to `checkHtmlElement`:

```js
if (element !== null && element !== undefined) {
  checkHtmlElement(element, ...)
}
```

Reply:
That is a solid plan! I will aim to update tonight!